templates/docs/upgrade-guide-v3.md

```diff
@@ -66,3 +66,5 @@ Use of `.col` classes outside of `.row` is deprecated. If you use `.col-X` class
 ### Tables
 
 We renamed and deprecated `p-table-expanding` and `p-table-expanding__panel`. Use `p-table--expanding` and `p-table__expanding-panel` instead.
+
+We removed the `p-table--sortable` that was previously required to enable sorting functionality in the tables. Currently any table with correctly used `aria-sort` attributes on column headers can be sorted. The `p-table--sortable` class name can be removed from HTML (any relevant JavaScript may need to be updated).
```

Comment:
:scream: I was not aware of this guide! I think there may be one or two things I've deprecated that should be mentioned here (code copyable for instance)

Reply:
@sowasred2012 No worries, I tend to forget about it as well. when you have a sec give it a look - I may have added what was missing in the meantime.
If not, just go around and compare 'deprecated' list from component status and check if it's mentioned there.

It doesn't have to be a final nicely edited guide yet, but a draft so we have everything in place when we need it.